src/SegmentedArc.js

```diff
@@ -7,6 +7,7 @@ import Segment from './components/Segment';
 import Cap from './components/Cap';
 import RangesDisplay from './components/RangesDisplay';
 
+
```

Comment:
Did we mean to add this space?

Reply:
Oops, nope. I was going to import something and then changed my mind but accidentally left the space. Good catch!

Reply:
Fixing